src/modules/drawing/components/DrawingExporter.vue

```diff
@@ -1,11 +1,12 @@
 <template>
     <DropdownButton
-        class="m-1"
+        class="m-1 drawing-button border-right"
```

Comment:
As these classes are defined in the parent module (and also scoped only to the parent module), I think you should also assign them in the parent class, as else it breaks the independence of the modules and is harder to understand. I.e. like this:
`<DrawingExporter :is-drawing-empty="isDrawingEmpty" class="drawing-button border-right" />`

Reply:
This seems like a good idea. It is now done.